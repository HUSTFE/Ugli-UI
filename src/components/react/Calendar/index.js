/* eslint-disable */
import React, { Component } from 'react'
import PropTypes from 'prop-types'

import styles from '@style/Calendar/index.sass'

class Calendar extends Component {
  render() {
    return <div>Calendar</div>
  }
}

export default Calendar
