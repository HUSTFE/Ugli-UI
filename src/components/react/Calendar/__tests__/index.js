import React from 'react'
import renderer from 'react-test-renderer'
import Calendar from '../index'

describe('Calendar', () => {
  it('snake spinner render without crash', () => {
    const component = renderer.create(<Calendar />)
    const json = component.toJSON()
    expect(json).toMatchSnapshot()
  })
})
