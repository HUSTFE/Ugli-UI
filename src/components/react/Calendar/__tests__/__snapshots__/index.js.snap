// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar snake spinner render without crash 1`] = `
<div>
  Calendar
</div>
`;
